### DISCUSSION TITLE
Is it possible to integrate code-server with a desktop application (such as Visual Studio Code/VS Code)?

### DISCUSSION THREAD

**Discussion by awildboop** [coder/code-server#2786] [Q&A] [answered]
I personally prefer to use the desktop version of VS Code - however, I like the versatility of code-server. Would it be possible to use both code-server and a desktop application (such as vscode itself)?

Potentially, could I just use VS Code's SSH feature and have it install itself on the same machine as code-server, then just use the same directory for development?

**Comment by awildboop** [comment, marked as answer]
So I decided to take a risk and use VS Code's SSH extension to try it, and it seems to be working okay. code-server does not seem to be experiencing any issues with it, nor does the desktop version of VS Code. I'll update the question title to improve SEO for similar questions, and will update this answer if I encounter any problems using both.

**Reply by code-asher** [reply]
Yeah that should work perfectly! They'll operate completely independently.
